.github/workflows/pr_build.yaml

```diff
@@ -62,6 +74,7 @@ jobs:
         run: |
           docker tag ghcr.io/spiffe/spire-controller-manager:devel ghcr.io/spiffe/spire-controller-manager:nightly
           (cd demo; ./test.sh)
+
   success:
     needs: [unit-test, build-image, test-image]
```

Comment:
we should add lint as a dependency for success

Reply:
done